src/courseware/course/bookmark/BookmarkButton.test.jsx

```diff
@@ -44,7 +44,7 @@ describe('Bookmark Button', () => {
   it('handles adding bookmark', async () => {
     render(<BookmarkButton {...mockData} />);
 
-    const button = screen.getByRole('button', { name: 'Bookmark this page' });
+    const button = screen.getByRole('button', { name: 'Bookmark Bookmark this page' });
```

Comment:
Yes, RTL searches for all names that are inside the button. This is expected because we replaced the FontAwesome icons that did not provide names for screen readers with Paragon icons that do.
In this example, we have the name of the icon and the inner text of the button.